public/assets/js/main.js

```diff
@@ -1,5 +1,6 @@
 /* eslint-env browser */
 /* exported tryIt, toggleCode */
+'use strict';
```

Comment:
This isn't right BTW, I will take care of it tomorrow with stricter options too.